### ISSUE TITLE
Error in getOption("SeuratDisk.dtypes.dataframe_as_group", x = TRUE)

### ISSUE THREAD

**Issue by Acribbs** [mojaveazure/seurat-disk#4]
Just testing the functionality and have encountered the following error:

```
Convert("kallisto.dir/HEK293-NIH3T3-DropSeq2_S2/bus/counts_unfiltered/adata.h5ad", dest = "h5seurat", assay="RNA", overwrite = TRUE)
Unknown file type: h5adCreating h5Seurat file for version 3.1.2
Adding X as data
Adding X as counts
Adding meta.features from var
Error in getOption("SeuratDisk.dtypes.dataframe_as_group", x = TRUE) : 
  'x' must be a character string
```

Any advice would be much appreciated Thanks

**Comment by Acribbs** [comment]
Scanpy and ann data: `scanpy==1.4.5 anndata==0.7.1`

sessionInfo()
```
R version 4.0.0 (2020-04-24)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] SeuratDisk_0.0.0.9008 forcats_0.5.0         dplyr_0.8.5           purrr_0.3.4           readr_1.3.1           tidyr_1.1.0          
 [7] tibble_3.0.1          ggplot2_3.3.0         tidyverse_1.3.0       Seurat_3.1.5          stringr_1.4.0        

loaded via a namespace (and not attached):
 [1] nlme_3.1-147       tsne_0.1-3         fs_1.4.1           bit64_0.9-7        lubridate_1.7.8    RcppAnnoy_0.0.16   RColorBrewer_1.1-2
 [8] httr_1.4.1         sctransform_0.2.1  tools_4.0.0        backports_1.1.7    R6_2.4.1           irlba_2.3.3        KernSmooth_2.23-17
[15] uwot_0.1.8         DBI_1.1.0          lazyeval_0.2.2     colorspace_1.4-1   withr_2.2.0        tidyselect_1.1.0   gridExtra_2.3     
[22] bit_1.1-15.2       compiler_4.0.0     cli_2.0.2          rvest_0.3.5        hdf5r_1.3.2        xml2_1.3.2         plotly_4.9.2.1    
[29] scales_1.1.1       lmtest_0.9-37      ggridges_0.5.2     pbapply_1.4-2      digest_0.6.25      pkgconfig_2.0.3    htmltools_0.4.0   
[36] dbplyr_1.4.3       htmlwidgets_1.5.1  rlang_0.4.6        readxl_1.3.1       rstudioapi_0.11    generics_0.0.2     zoo_1.8-8         
[43] jsonlite_1.6.1     ica_1.0-2          magrittr_1.5       patchwork_1.0.0    Matrix_1.2-18      fansi_0.4.1        Rcpp_1.0.4.6      
[50] munsell_0.5.0      ape_5.3            reticulate_1.15    lifecycle_0.2.0    stringi_1.4.6      MASS_7.3-51.6      Rtsne_0.15        
[57] plyr_1.8.6         grid_4.0.0         parallel_4.0.0     listenv_0.8.0      ggrepel_0.8.2      crayon_1.3.4       lattice_0.20-41   
[64] haven_2.2.0        cowplot_1.0.0      splines_4.0.0      hms_0.5.3          knitr_1.28         pillar_1.4.4       igraph_1.2.5      
[71] future.apply_1.5.0 reshape2_1.4.4     codetools_0.2-16   leiden_0.3.3       reprex_0.3.0       glue_1.4.1         packrat_0.5.0     
[78] data.table_1.12.8  modelr_0.1.7       png_0.1-7          vctrs_0.3.0        cellranger_1.1.0   gtable_0.3.0       RANN_2.6.1        
[85] future_1.17.0      assertthat_0.2.1   xfun_0.13          rsvd_1.0.3         broom_0.5.6        survival_3.1-12    viridisLite_0.3.0 
[92] cluster_2.1.0      globals_0.12.5     fitdistrplus_1.1-1 ellipsis_0.3.1     ROCR_1.0-11  
```

**Comment by mojaveazure** [comment]
Hi Adam,

This should be fixed thanks to #3 and available on the `master` branch
